contrib/gin-gonic/gin/gintrace.go

```diff
@@ -30,7 +30,8 @@ func init() {
 
 // Middleware returns middleware that will trace incoming requests. If service is empty then the
 // default service name will be used.
-func Middleware(service string, opts ...Option) gin.HandlerFunc {
+func Middleware(opts ...Option) gin.HandlerFunc {
```

Comment:
That looks like a breaking change? There could be existing usage that passes `service` and now it is no longer legal to do that... Proof being that it appears to break the associated tests...

Reply:
This was only done to check if things really breaks we do something rhat should break, I will revert it